package.json

```diff
@@ -8,8 +8,9 @@
     "url": "https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite.git"
   },
   "scripts": {
+    "postinstall": "pnpm i rimraf -g",
```

Comment:
Fundamentally, I question whether the ability to clear the node modules folder is a feature that should be added, even at the expense of adding an implicit rimraf dependency.

Reply:
@Jonghakseo That's your decision, we can merge this PR or remove this command completely.

Reply:
My obvious solution is to remove the script. It's not a core part of the Chrome extension development, so it's better to remove it than to create noise.

Reply:
@Jonghakseo It's maybe useful sometimes, but yeah it's not necessary.